Problem2.java

```diff
@@ -0,0 +1,55 @@
+
+/**
+Leet Code Submitted : YES
+Space Complexity : O(1)
+Time Complexity : O(N^2)
+
```

Comment:
How is your time complexity O(n^2)

Reply:
Yeah every element is checked only once. So, it should be O(N)

Reply:
it will be O(nlogn).You are sorting also, Take care while calculating complexity by algorithm, code will confuse you

Reply:
The time complexity given by you is the correct one. I misunderstood  it a little.

Reply:
Prateek, the complexity given by you is the correct one. I misunderstood it.